integration-tests/smoke/vrfv2plus_test.go

```diff
@@ -785,7 +787,61 @@ func TestVRFv2PlusMigration(t *testing.T) {
 	)
 	require.NoError(t, err, "error setting up VRF v2_5 env")
 
+	t.Run("Test migration of Subscription Billing subID", func(t *testing.T) {
+		// Migrate subscription from old coordinator to new coordinator, verify if balances
+		// are moved correctly and requests can be made successfully in the subscription in
+		// new coordinator
+		testSubMigration(subIDs, false, vrfv2PlusContracts, t, l, env, vrfv2PlusData, config,
+			linkAddress, mockETHLinkFeedAddress, nodesMap)
+	})
+
+	t.Run("Test migration of direct billing using VRFV2PlusWrapper subID", func(t *testing.T) {
+		// Migrate wrapper subscription from old coordinator to new coordinator, verify if balances
+		// are moved correctly and requests can be made successfully in the subscription in
+		// new coordinator
+		testSubMigration(subIDs, true, vrfv2PlusContracts, t, l, env, vrfv2PlusData, config,
+			linkAddress, mockETHLinkFeedAddress, nodesMap)
+	})
+}
+
+func testSubMigration(subIDs []*big.Int,
+	isDirectBilling bool,
+	vrfv2PlusContracts *vrfcommon.VRFContracts,
+	t *testing.T,
+	l zerolog.Logger,
+	env *test_env.CLClusterTestEnv,
+	vrfv2PlusData *vrfcommon.VRFKeyData,
+	config tc.TestConfig,
+	linkAddress contracts.LinkToken,
+	mockETHLinkFeedAddress contracts.MockETHLINKFeed,
+	nodesMap map[vrfcommon.VRFNodeType]*vrfcommon.VRFNode) {
+
 	subID := subIDs[0]
+	var err error
+	var wrapperContracts *vrfv2plus.VRFV2PlusWrapperContracts
+	configCopy := config.MustCopy().(tc.TestConfig)
+	expectedNumSubs := 1
+	if isDirectBilling {
```

Comment:
so this `if` adds additional burden and looks quite "smelly"

Reply:
Removed common function between normal migration and direct funding migration